pkg/infradb/infradb.go

```diff
@@ -321,24 +321,25 @@ func UpdateLBStatus(name string, resourceVersion string, notificationID string,
 
 			// Delete VNI from the VPN map
 			vpns := make(map[uint32]bool)
-			found, err = infradb.client.Get("vpns", &vpns)
-			if err != nil {
-				log.Println(err)
-				return err
-			}
-			if !found {
-				log.Println("UpdateLBStatus(): No VPNs have been found")
-				return ErrKeyNotFound
-			}
 			if lb.Spec.Vni != nil {
-				delete(vpns, *lb.Spec.Vni)
-			}
-			err = infradb.client.Set("vpns", &vpns)
-			if err != nil {
-				log.Println(err)
-				return err
+				found, err = infradb.client.Get("vpns", &vpns)
+				if err != nil {
+					log.Println(err)
```

Comment:
is it a good strategy to log an error here and return it?
Apparently, a caller will likely also print the same error. Doing so, we are causing a lot of noise in the application logs for, in my opinion, little value.

Reply:
done